types/mapbox-gl/index.d.ts

```diff
@@ -35,7 +39,7 @@ declare namespace mapboxgl {
 
 		getBounds(): mapboxgl.LngLatBounds;
 
-		setMaxBounds(lnglatbounds?: mapboxgl.LngLatBounds | number[][]): this;
+		setMaxBounds(lnglatbounds?: LngLatBoundsLike): this;
```

Comment:
Yep. By better defining the types it helps ensure all options are available for input